src/components/VotePanel.tsx

```diff
@@ -50,6 +53,32 @@ function Eligibility({
   }
 }
 
+function VoteOnQuestions(props: {
+  ecOfferId: number;
+  instance?: [property: string, value: RpcRemote][];
+}) {
+  const history = usePublishedHistory(
+    'committees.Economic_Committee.latestQuestion'
+  );
+  console.debug('question history', history.data);
+
+  if (history.status !== 'received') {
+    return <p>{capitalize(history.status)} for a question...</p>;
+  }
+
+  return (
+    <>
+      {history.data.map((details: QuestionDetails) => (
```

Comment:
Maybe we can just filter questions whose deadlines have passed? Or at least make sure we sort the open questions to the top of the list.

Reply:
Great idea! I'll amend with that.